frontend/package.json

```diff
@@ -37,8 +38,8 @@
     "start": "npm-run-all -p watch-css start-js",
     "start-local": "REACT_APP_AWS_ENDPOINT='http://localhost:5000/graphql' yarn run start",
     "validate-map-style": "node src/Map/style/scripts/validate.js",
-    "build-map-style": "yarn build-map-icons; node src/Map/style/scripts/build.js",
-    "build-map-icons": "spritezero public/mapboxstyle/sprite src/Map/style/icons/; spritezero --retina public/mapboxstyle/sprite@2x src/Map/style/icons/",
+    "build-map-style": "yarn build-map-icons; node src/components/Map/style/scripts/build.js",
+    "build-map-icons": "spritezero public/mapboxstyle/sprite src/components/Map/style/icons/; spritezero --retina public/mapboxstyle/sprite@2x src/components/Map/style/icons/",
```

Comment:
Note these changes I made to the map build scripts. They were erroring on me bc they couldn't find the files from src. Think this was just missed when we moved a bunch of stuff into the `components` dir